msg/SensorWind.msg

```diff
@@ -0,0 +1,5 @@
+uint64 timestamp		# time since system start (microseconds)
+uint32 device_id                # unique device ID for the sensor that does not change between power cycles
+float32 wind_speed		# the speed being reported by the wind sensor
+float32 wind_direction		# the direction bein report by the wind sensor
```

Comment:
Isn't this measuring the relative air direction rather than wind?

Reply:
It is measuring the relative airflow.

Reply:
IMHO, it would be better to rename the fields so that it doesnt get confused with the actual wind estimates

Reply:
Happy to change it. Do you have a suggestion?

Mine would be speed and direction, and then perhaps rename  the uorb topic to sensor_airflow?

Reply:
Sounds good on myside!